code/game/machinery/telecomms/computers/logbrowser.dm

```diff
@@ -3,6 +3,7 @@
 /obj/machinery/computer/telecomms/server
 	name = "telecommunications server monitoring console"
 	icon_screen = "comm_logs"
+	desc = "Has full access to all details and record of the telecommunications network it's monitoring."
```

Comment:
"Has full access to all details and record__s__ of the telecommunications network it's monitoring." ?